src/gc/gc.cpp

```diff
@@ -5052,6 +5055,14 @@ extern "C" uint64_t __rdtsc();
         // all buffer access times being reported as equal in access_time().
         return 0;
     }
+#elif defined(_TARGET_WASM_)
```

Comment:
It would be better to just put the dummy implementation into the `#else` instead.

Reply:
Shall i collapse `_TARGET_ARM_`, `_TARGET_ARM64_` and `_TARGET_WASM_` (that all have identical implementations, just different comments) into `#else`?

Reply:
Yes, I think so.

Reply:
yeah please do

---
In reply to: [292639177](https://github.com/dotnet/coreclr/pull/25092#discussion_r292639177) [](ancestors = 292639177)